docs/installation/gke.md

```diff
@@ -51,16 +54,9 @@ helm repo update
 Install KubeAI with [Helm](https://helm.sh/docs/intro/install/).
 
 ```bash
-cat <<EOF > kubeai.yaml
-resourceProfiles:
-  nvidia-gpu-l4:
-    nodeSelector:
-      cloud.google.com/gke-accelerator: "nvidia-l4"
-      cloud.google.com/gke-spot: "true"
-EOF
-
+curl -L -O https://raw.githubusercontent.com/substratusai/kubeai/refs/heads/main/charts/kubeai/values-gke.yaml
```

Comment:
Prefer to avoid the `curl` by updating the helm flag `-f https://raw.githubusercontent.com/substratusai/kubeai/refs/heads/main/charts/kubeai/values-gke.yaml`

Reply:
I started with that, but the url is too long and I felt like it hides away too much then. I prefer the curl so people can easily edit later and have it saved somewhere.

Reply:
Sounds good